_posts/2020-12-09-coiled-webinar-post-event-writeup.md

```diff
@@ -0,0 +1,152 @@
+
+Thanks to Florian’s comprehensive demo, after reading this post, you’ll know:
+
+* That buying milk at your local supermarket is a highly non-trivial process,
+* How to incorporate a relational database into your data pipeline,
+* How to build a ML data pipeline at terabyte scale using Parquet, Dask, and Kartothek,
+* How to join data at scale without breaking a sweat (kind of),
+* Where to go next with resources on Dask, supply chains, and more.
+
+Thank you to Florian and his colleague [Sebastian Neubauer](https://www.linkedin.com/in/sebastian-neubauer), 
+Senior Data Scientist at Blue Yonder, who also joined us for this live stream, for the feedback on an earlier draft of 
+this article!
+
+# Blue Yonder’s approach to the problem of supply chain management 
+
+>“We need to crunch this together, ideally in a way that is cheap to store and in a way that data scientists can 
+scale-out, and this is where Dask comes in.” 
+
+First things first, we were curious to know how many clusters Blue Yonder uses. Florian said, “We have a lot of 
+customers and different environments, this all totals to around 700 clusters at the moment.” 200 of those clusters are 
+production use cases, which means they’re fully automated with no human involvement. 
+
+Before we jumped into the demonstration, Florian gave us a lesson on supply chain networks, outlined through the 
+example of a grocery store buying milk. When thinking about supply chain fulfillment, two questions emerge: 
+
+1) The prediction: What is the demand? (which turns out to be a huge machine learning problem), and 
+
+2) The decision: What’s the optimal amount we should order, considering the current state of the supply chain? 
+
+Blue Yonder approaches the concept of supply chain management by breaking it down into pieces. A relatively simple 
+example could be: store level demand forecast (ML), store/distribution center network optimization, vendor ordering, 
+and truckload, as seen in the graph below.
+ 
+![supply chain network](/assets/images/2020-12-09-coiled-webinar-post-event4.png)
+
+This an ongoing and iterating process. Florian noted that “the number of feedback loops involved makes this a highly 
+non-trivial problem”. So, your glass of milk is more complicated than it looks!
+
+We then jumped into a demonstration in which Florian walked us through an example of building out a ML data pipeline 
+for a mid-sized vendor. We saw how quickly this involves 10 million time series. You can follow along with this 
+[notebook](https://github.com/fjetter/coiled-kartothek-demo).
+
+![supply chain network](/assets/images/2020-12-09-coiled-webinar-post-event5.png)
+
+Using 10 years years of data from the customer, Blue Yonder is working with 40 terabytes of uncompressed data, which 
```

Comment:
"years years" redundant.
Use: "Blue Yonder has been working" instead of "Blue Yonder is working"

Reply:
fixed